src/Microsoft.AspNet.Mvc.TagHelpers/ScriptTagHelper.cs

```diff
@@ -307,6 +306,44 @@ private void BuildFallbackBlock(TagHelperAttributeList attributes, DefaultTagHel
             }
         }
 
+        private void BuildScriptTag(
```

Comment:
Move this back down where the `BuildScriptTag()` method was before. That way I'll be able to see what changed here.